packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -278,6 +278,36 @@ abstract class WebViewPlatformController {
     throw UnimplementedError(
         "WebView getTitle is not implemented on the current platform");
   }
+
+  /// Set the scrolled position of this view.
+  /// 'x' - the x position to scroll to.
```

Comment:
Document the unit here as well, this documentation may be the only thing a platform implementor is reading.

Reply:
Same below

Reply:
Added in incoming commit.